pkg/kubelet/volumemanager/populator/desired_state_of_world_populator.go

```diff
@@ -188,7 +196,7 @@ func (dswp *desiredStateOfWorldPopulator) isPodTerminated(pod *v1.Pod) bool {
 
 // Iterate through all pods and add to desired state of world if they don't
 // exist but should
-func (dswp *desiredStateOfWorldPopulator) findAndAddNewPods() {
+func (dswp *desiredStateOfWorldPopulator) findAndAddNewPods() bool {
```

Comment:
Is this bool return value only used for unit testing? I prefer trying some other way in the unit test to verify the behavior, such as checking the dsw cache.

Reply:
The cache is checked in test.
I can drop the return value.